modules/permission-sets/variables.tf

```diff
@@ -11,6 +11,11 @@ variable "permission_sets" {
       name = string
       path = string
     }))
+    permissions_boundary_attachment = object({
```

Comment:
Hi @destornillador,

Thank you for your contribution!

Could you perhaps make the `permissions_boundary_attachment` optional as well? Then your addition will be fully backwards compatible. 

Thank you very much!

Reply:
Hey @JoeriMalmberg,

Thank you for your fast response. Already made the change you suggested. What do you think?

Reply:
I think that's great!

Thank you very much